client/src/lifecycleManager.js

```diff
@@ -129,15 +131,15 @@ class LifecycleManager extends LuigiClientBase {
   }
 
   /**
-   * Registers a listener called with the context object as soon as Luigi is instantiated. Defer your application bootstrap if you depend on authentication data coming from Luigi.
+   * Registers a listener called with the context object and the Luigi core domain as soon as Luigi is instantiated. Defer your application bootstrap if you depend on authentication data coming from Luigi.
```

Comment:
Should we also call the context update listeners with the Luigi Core domain for consistency?

Reply:
do we need it in context update listeners? I don't think so.